mini-cloudinary.rb

```diff
@@ -16,56 +16,57 @@ def parse_error_to_json(code, message)
         }
         JSON[json]
     end
+
+    def build_error_array(code, message)
+        [code, parse_error_to_json(code, message)]
+    end
 end
 
 class MiniCloudinary
 
-    include ErrorHandler
-
     def initialize(path)
         @local_path = path
     end
 
-    def image_doesnt_exist?(url)
-        
-        uri = URI.parse(url)
-        request = Net::HTTP.new(uri.host, uri.port)
-        request.use_ssl = (uri.scheme == 'https')
-        response = request.request_head(uri.path)
-        response.code.to_i != STATUS_OK
+    def resize_with_black_background(image, width, height)
```

Comment:
1. perhaps `resize_with_background` makes more sense. It opens up the possibility to add an optional color.
2. I'd consider using named parameters here